### PR TITLE
[Inji-604]: Addressing the issue of activity log and data back up being called again during app launch and sharing.

### PR DESCRIPTION
**Pull request by balachandarg-tw**
Fix:
- Removing Verification call during Refresh and adding it only after downloading VC.

### CODE REVIEW

File: machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts [hunk 1]
```diff
@@ -89,7 +89,6 @@ const model = createModel(
       DOWNLOAD_READY: () => ({}),
       FAILED: () => ({}),
       GET_VC_RESPONSE: (vc: VC) => ({vc}),
-      VERIFY: () => ({}),
```
Comment (Alka1703) [inline machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts:92]: Why is this VERIFY event from here?

**Review by Alka1703 — CHANGES_REQUESTED** [review]

File: machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts (hunk 1 above)
Comment (balachandarg-tw) [reply machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts:92]: This event is unused and might have been used for any past features before.

Comment (Alka1703) [reply machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts:92]: Can you remove it from EsignetMosipVCItemMachine as well?

Comment (balachandarg-tw) [reply machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts:92]: Removed the event.

**Review by Alka1703 — APPROVED** [review]
LGTM

**Review by swatigoel — APPROVED** [review]

**Pull request merged**